tests/e2e/redis/basic-ingest-redis-serving.py

```diff
@@ -119,13 +123,29 @@ def test_basic_register_feature_set_success(client):
     assert driver_fs_actual == driver_fs_expected
 
     # Register feature set with project
-    cust_trans_fs_expected = FeatureSet.from_yaml(f"{DIR_PATH}/basic/cust_trans_fs.yaml")
+    cust_trans_fs_expected = FeatureSet.from_yaml(
+        f"{DIR_PATH}/basic/cust_trans_fs.yaml"
+    )
     client.set_project(PROJECT_NAME)
     client.apply(cust_trans_fs_expected)
-    cust_trans_fs_actual = client.get_feature_set("customer_transactions",
-                                                  project=PROJECT_NAME)
+    cust_trans_fs_actual = client.get_feature_set(
+        "customer_transactions", project=PROJECT_NAME
+    )
     assert cust_trans_fs_actual == cust_trans_fs_expected
 
+    # Register feature set with labels
+    driver_fs_expected = FeatureSet.from_yaml(f"{DIR_PATH}/basic/driver_fs.yaml")
```

Comment:
This test is too brittle. I think it would be better to move all of the logic into code instead of using the YAML files. Otherwise if somebody modifies driver_fs then this test will break.

Reply:
Updated.